test/testcafe/framework/page-objects/IdPAuthenticatorPageObject.js

```diff
@@ -0,0 +1,29 @@
+import { ClientFunction } from 'testcafe';
+import BasePageObject from './BasePageObject';
+
+export default class IdPAuthenticatorPageObject extends BasePageObject {
+  constructor (t) {
+    super(t);
+  }
+
+  async getPageUrl() {
+    const pageUrl = await ClientFunction(() => window.location.href)();
+    return pageUrl;
+  }
+
+  getPageTitle() {
+    return this.form.getElement('.okta-form-title').textContent;
+  }
+
+  getPageSubtitle() {
+    return this.form.getElement('.okta-form-subtitle').textContent;
+  }
+
+  enroll() {
```

Comment:
nit: `clickSave` or `submitForm` might be reusable instead of having `enroll` and `verify`? Or maybe `clickEnrollButton` and `clickVerifyButton` would be similar to what we currently have in other page objects?